chromium_src/chrome/tools/build/win/create_installer_archive.py

```diff
@@ -98,6 +107,8 @@ def CopyExtensionLocalization(extension_name, locales_src_dir_path, config,
     locales_dest_path = os.path.realpath(locales_dest_path)
     shutil.rmtree(locales_dest_path, ignore_errors=True)
     shutil.copytree(locales_src_dir_path, locales_dest_path)
+    os.rename(os.path.join(locales_dest_path, 'no'),
```

Comment:
a comment?

Reply:
I added comments in 06f36a9260fc02c25901946eb295af63f3f36d21.

@mkarolin would you say this comment is accurate?

```
+      # The "nb" locale is a subset of "no". Chromium uses the former, while
+      # Transifex uses the latter. To integrate with Transifex, our code renames
+      # "nb" to "no". The following creates a bridge between the two paradigms:
       source_locale = string_replace(locale, "nb", "no")
```